pxr/imaging/hio/image.h

```diff
@@ -199,10 +199,13 @@ HioImage::GetMetadata(TfToken const & key, T * value) const
     return true;
 }
 
+ARCH_PRAGMA_PUSH
+ARCH_PRAGMA_NON_DLL_INTERFACE_CLASS_USED_AS_BASE_FOR_DLL_INTERFACE_CLASS
```

Comment:
same issue, no symbols exported., is the proper fix to remove HIO_API here?

Reply:
Potentially - I do not have sufficient understanding of the code base to make that call.

Reply:
Could you test removing the decoration here to see if it resolves the warning?

Reply:
Will do.

Reply:
Same as above.